Code/GraphMol/catch_graphmol.cpp

```diff
@@ -1486,3 +1486,61 @@ TEST_CASE("github #2890", "[bug][molops][stereo]") {
     CHECK(bond->getStereo() == Bond::STEREOANY);
     CHECK(bond->getStereoAtoms().size() == 2);
 }
+
+
+TEST_CASE("github #3150 MolOps::removeHs removes hydrides", "[bug][molops]") {
```

Comment:
The tests here are good, but I'd like to also see what happens in each case when `ps.removeHydrides` is `true`.